extensions/resource-deployment/src/ui/toolsAndEulaSettingsPage.ts

```diff
@@ -302,7 +323,7 @@ export class ToolsAndEulaPage extends ResourceTypePage {
 					// Update the informational message
 					this.wizard.wizardObject.message = {
 						level: azdata.window.MessageLevel.Information,
-						text: localize('deploymentDialog.InstallingTool', "Required tool '{0}' [ {1} ] is being installed now.", tool.displayName, tool.homePage)
+						text: loc.getToolInstallingMessage(tool)
 					};
 					await this._tools[i].install();
```

Comment:
It doesn't necessarily have to be part of this PR - but it would be great to have the "Installing" status using the "withProgress" function from VS Code in a notification (toast) - see https://github.com/microsoft/azuredatastudio/blob/main/extensions%2Farc%2Fsrc%2Fui%2Fdashboards%2Fmiaa%2FmiaaComputeAndStoragePage.ts#L125 for an example

Right now it's not really easy to see that it's doing something and if we're allowing the user to cancel out of this dialog it would still be good to have the notification running so they know the tool is still being installed.